examples/catboost_simple.py

```diff
@@ -0,0 +1,66 @@
+"""
+Optuna example that optimizes a classifier configuration for cancer dataset using
+Catboost.
+
+In this example, we optimize the validation accuracy of cancer detection using
+Catboost. We optimize both the choice of booster model and their hyperparameters.
+
+We have following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python catboost_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize catboost_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+import catboost as cb
+import numpy as np
+import sklearn.datasets
+import sklearn.metrics
+from sklearn.model_selection import train_test_split
+
+import optuna
+
+
+def objective(trial):
+    data, target = sklearn.datasets.load_breast_cancer(return_X_y=True)
+    train_x, test_x, train_y, test_y = train_test_split(data, target, test_size=0.3)
+
+    param = {
+        'objective': trial.suggest_categorical('objective', ['Logloss', 'CrossEntropy']),
+        'iterations': trial.suggest_int('num_leaves', 500, 2000),
```

Comment:
`num_leaves` seems a typo.
```suggestion
        'iterations': trial.suggest_int('iterations', 500, 2000),
```

Reply:
In addition, I think that tuning of `iterations` parameters can be eliminated.
Usually, IMO, the combination of 'setting sufficiently large (fixed) `iterations` value' and 'enabling early stopping' archives a good result with less time compared to tuning `iterations` parameter.